rewrite-java/src/main/java/org/openrewrite/java/internal/beta/CallMatcher.java

```diff
@@ -0,0 +1,133 @@
+package org.openrewrite.java.internal.beta;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import org.openrewrite.Cursor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.Flag;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@FunctionalInterface
```

Comment:
needs an `@Incubating(since = "7.25.0")

Reply:
It's in a package called `org.openrewrite.java.internal.beta`, but sure. Nobody should be using this API outside of the internal consumers